src/content/en/updates/2017/02/chrome-57-deprecations.md

```diff
@@ -0,0 +1,121 @@
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5688905986736128) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=678547)
+
+## Remove BluetoothDevice.uuids attribute
+
+The `BluetoothDevice.uuids` attribute is being removed to bring the
+[Bluetooth API](https://www.chromestatus.com/features/5264933985976320) in
+line with the current specification. You can retrieve a UUID by calling
+[device.watchAdvertisements()](https://webbluetoothcg.github.io/web-bluetooth/#dom-bluetoothdevice-watchadvertisements).
+
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=653317&desc=2)
+
+## Remove key generation element
+
+Since Chrome 49, &lt;keygen&gt;'s default behaviour has been to return the empty string, unless a permission was granted to this page. IE/Edge do not support &lt;keygen&gt; and have not indicated public signals to support &lt;keygen&gt;. Firefox already gates &lt;keygen&gt; behind a user gesture, but is publicly supportive of removing it. Safari ships &lt;keygen&gt; and has not expressed public views regarding its continued support. With Chrome 57, this element is removed.
+
+[Intent to Remove](https://groups.google.com/a/chromium.org/d/topic/blink-dev/pX5NbX0Xack/discussion) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5716060992962560) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=568184)
+
+## Remove webkit-prefixed IndexedDB global aliases
+
+The `IndexedDB` entry point and global constructors were exposed with `webkit` prefixes somewhere around Chrome 11. The non-prefixed versions were added in Chrome 24 and the prefixed versions were deprecated in Chrome 38. The following interfaces are affected:
+
+* `webkitIndexedDB` (main entry point)
+* `webkitIDBKeyRange` (non-callable global constructor, but has useful static methods)
+* `webkitIDBCursor`
+* `webkitIDBDatabase`
+* `webkitIDBFactory`
+* `webkitIDBIndex`
+* `webkitIDBObjectStore`
+* `webkitIDBRequest`
+* `webkitIDBTransaction` (non-callable global constructors)
+
+[Intent to Remove]() &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5775330191081472) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=665243)
+
+## WebAudio: Remove prefixed AudioContext and OfflineAudioContext
+
+Chrome has supported `WebAudio` since mid 2011, including `AudioContext`. `OfflineAudioContext` was added the following year. Given how long the standard interfaces and Googles long-term goal of removing prefixed features, the prefixed versions of these interfaces have been deprecated since late 2014 and are now being removed.
+
+[Intent to Remove](https://groups.google.com/a/chromium.org/d/topic/blink-dev/of6S04dUf54/discussion) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/4571020824412160) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=665887)
+
+## Deprecation policy
```

Comment:
Is this next bit standard, added to every removals & deprecations post? If so, we should move it to a widget. We can do that after this is posted.